src/Search/Search.js

```diff
@@ -91,6 +91,8 @@ class Search extends React.Component {
 
     const newFilters = [];
 
+    // will be fixed in the future
```

Comment:
Please try the following in order to get rid of these comments. As I see eslint is wrong about these warnings. According to [comment](https://github.com/eslint/eslint/issues/12117#issuecomment-533991195) upgrade of babel-eslint should do the trick.
`"babel-eslint": "^10.0.3"`